asm/macros/battle_anim.inc

```diff
@@ -1,11 +1,11 @@
-	.macro loadsprite id
+	.macro loadsprite currentFuncId
 	.byte 0x00
-	.2byte \id
+	.2byte \currentFuncId
```

Comment:
I know this is a draft, but in case it gets missed later: a lot of these `id` -> `currentFuncId` look accidental.